galaxy/Dockerfile

```diff
@@ -125,6 +125,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars galaxy_extras_config_condor=True \
     --extra-vars galaxy_extras_config_condor_docker=True \
     --extra-vars galaxy_extras_config_rabbitmq=True \
+    --extra-vars galaxy_extras_config_cvmfs=True \
```

Comment:
I still don't get why is the default from galaxy-extras not carrying over but setting `GALAXY_CONFIG_TOOL_DATA_TABLE_CONFIG_PATH=/etc/galaxy/tool_data_table_conf.xml` as an env var at docker start time makes this work out of the box. So I guess it can be set as the default in this file instead under the `ENV` section.